app/views/contact/govuk/accessible_format_requests/contact_details.html.erb

```diff
@@ -0,0 +1,63 @@
+<% content_for :title do %>
+  <% if flash[:input_errors]&.any? %>
+    <%= t("controllers.contact.govuk.accessible_format_requests.error_summary") %> -
+  <% end %>
+  <%= t("controllers.contact.govuk.accessible_format_requests.contact_information_heading") +
+    " - " + t("controllers.contact.govuk.accessible_format_requests.caption")
+  %>
+<% end %>
+
+<%= render "govuk_publishing_components/components/back_link", { href: @back_path } %>
+
+<% if flash[:input_errors]&.any? %>
+  <%= render "govuk_publishing_components/components/error_summary", {
+    id: "error-summary",
+    title: t("controllers.contact.govuk.accessible_format_requests.error_summary"),
+    items: flash[:input_errors].map { |err, href_target| { text: err, href: "\##{href_target}" } },
+  } %>
+<% end %>
+
+<%= form_with url: contact_govuk_request_accessible_format_send_request_path, method: :post, authenticity_token: false, html: { class: "contact-form", novalidate: "novalidate" } do |f| %>
```

Comment:
I think I just copied this over from @edwardkerry's original PR, so.. not sure. BUT: I don't think CSRF tokens will work here (they need a session cookie, which we don't have for feedback), so that might be the reason.